ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-97] Rehydration Pt. 2 Income shown in CYA

## Description of change
Add Income details to new Employment and Income Sections of CYA page
## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-97
## Notes for reviewer
Designs are presumed based on the design on interim CYAs [here](https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?type=design&node-id=8959%3A130668&mode=design&t=zL5fWCdmRelppqV4-1)
## Screenshots of changes (if applicable)

### Before changes:
**CYA**
<img width="891" alt="image" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/6b48fb28-0ab6-41f0-8811-36f1d48447d4">
**Print**
<img width="683" alt="image" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/093b1015-5f8f-420d-ba0c-1715c88480ed">

### After changes:
**CYA**
<img width="755" alt="image" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/12662ba6-47eb-4377-8fe9-b6c6a9712fb3">
**Print**
<img width="798" alt="image" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/419d6e42-b874-4879-b5c3-25ba0026d9f9">

## How to manually test the feature
1. Run bin/rails db:migrate RAILS_ENV=development
2. Create a new application, fill all details but DO NOT SUBMIT
3. Go to /steps/income/clients_income_before_tax
4. Select 'Yes'
5. Hit Save and continue
6. Reopen that application
7. Go to /steps/income/did_client_lose_job_being_in_custody
8. Hit Save and come back later
9. Reopen that application
10. Go to Review the application - check table correct and matches designs and copy
11. Submit application
12. Hit View completed application - check table correct and matches designs and copy
